src/filemanager.hpp

```diff
@@ -53,8 +53,8 @@ class FileManager {
 
   public:
     // "Low-level" interface.
-    static std::unique_ptr<File> open(PJ_CONTEXT *ctx, const char *filename,
-                                      FileAccess access);
+    static PROJ_DLL std::unique_ptr<File>
```

Comment:
thread synchronization concerns with this?

Reply:
I don't see any thread related issue here